i18n/zh-cn/docusaurus-plugin-content-docs/current/api/connect.md

```diff
@@ -68,15 +63,19 @@ const gearApi = await GearApi.create({ customTypes: { ...yourCustomTypesExample
 async function connect() {
   const gearApi = await GearApi.create();
 
-  const [chain, nodeName, nodeVersion] = await Promise.all([
+  const [chain, nodeName, version] = await Promise.all([
     gearApi.chain(),
     gearApi.nodeName(),
-    gearApi.nodeVersion(),
+    gearApi.version(),
   ]);
-  console.log(`You are connected to chain ${chain} using ${nodeName} v${nodeVersion}`);
-
-  gearApi.gearEvents.subscribeNewBlocks((header) => {
-    console.log(`New block with number: ${header.number.toNumber()} and hash: ${header.hash.toHex()}`);
+  console.log(
+    `You are connected to chain ${chain} using ${nodeName} v${version}`,
+  );
+
+  const unsub = await gearApi.gearEvents.subscribeNewBlocks((header) => {
```

Comment:
`subscribeNewBlocks` Revised to `subscribeToNewBlocks`
 you can try this  latest version code https://github.com/GearFans/example/blob/main/src/connect.ts

Reply:
> `subscribeNewBlocks` Revised to `subscribeToNewBlocks` you can try this latest version code https://github.com/GearFans/example/blob/main/src/connect.ts

Thanks for the comments. Pls help to try the EN version. CN version should align with the EN version.